src/pages/profiles/profile-creation.vue

```diff
@@ -565,7 +554,7 @@ export default {
           v-show="activeStepIndex > 0"
         )
         q-btn.q-px-xl.q-ml-sm(
-          :disable="submitting || !nextAvailable"
```

Comment:
Why this was removed?

Reply:
Because validation should happen after you click submit button, spoke with about that on testing session today with @leonieherma1

Reply:
Got it